doc/source/zuul.rst

```diff
@@ -65,5 +66,354 @@ each commit is correctly tested.
 Zuul's current status may be viewed at
 `<https://zuul.opendev.org/>`_.
```

Comment:
I know this one is not part of this PR, but this should probably point to https://zuul.otc-service.com